support/tests/test_tealium_helper.swift

```diff
@@ -30,27 +30,27 @@ enum TealiumTestError : Error {
 }
 
 let testStringArrayValue = ["value1", "value2"]
-let testOptionalData = [TealiumTestKey.stringKey: TealiumTestValue.stringValue as AnyObject,
-                        TealiumTestKey.stringArrayKey: testStringArrayValue as AnyObject] as [String : AnyObject]
+var testOptionalData = [TealiumTestKey.stringKey: TealiumTestValue.stringValue,
+                        TealiumTestKey.stringArrayKey: testStringArrayValue] as [String : Any]
 let testTealiumConfig = TealiumConfig(account:TealiumTestValue.account,
```

Comment:
Actually, no, the optionalData is a [String:Any] that is not persisted.  We would only be testing that Apple's [String:Any] work as expected, which would lead to a never-ending series of tests.